src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/UpdateInductionTest.kt

```diff
@@ -176,9 +176,9 @@ class UpdateInductionTest : IntegrationTestBase() {
       ),
       skillsAndInterests = aValidUpdateSkillsAndInterestsRequest(
         id = persistedInduction.skillsAndInterests!!.id!!,
-        skills = setOf(PersonalSkill.COMMUNICATION),
+        skills = setOf(PersonalSkillType.COMMUNICATION),
```

Comment:
I've renamed the enum `PersonalSkill` to `PersonalSkillType` to allow for a new `PersonalSkill` object.